utils/server-instance.js

```diff
@@ -31,6 +31,8 @@ class ServerInstance {
     this._app = express();
     this._app.use(cookieParser());
 
+    this._counter = 1;
```

Comment:
Could we just have this return Date.now() rather than have a counter and then have '/__echo/random/nocache/' ?

I'm just thinking it's less orchestrating, states we have no-cache header and "random" seems like it could be useful else where.